pkgs/native_assets_cli/lib/src/api/link_config.dart

```diff
@@ -31,6 +32,10 @@ abstract class LinkConfig implements HookConfig {
   /// `build.dart` script destined for this packages `link.dart`.
   Iterable<Asset> get assets;
 
+  /// The file containing the recorded usages, serialized by package:record_use.
+  @experimental
+  Uri? get recordedUsages;
```

Comment:
It's not really a Uri it's a filename, so why not use `String` for it?

(I understand that some people on the team prefer to use `Uri`s for everything but it's just confusing as this Uri cannot be a `http:...` uri, it has to be a file path - also most of our filepath abstractions (e.g. `package:path` as well as dart:io apis deal with strings)

Reply:
I was just trying to be consistent with the rest of `native_assets_cli`, which uses URIs. I am fine with either - URIs or strings and `package:path`.

Reply:
Yes, please be consistent in this package.

(@mkustermann I've stated this on previous PRs that might not have involved you. I prefer consistency. I could be convinced to switch to Strings instead of Uris with arguments for pros and cons. And if we switch, we should switch every Uri in this package.)

Reply:
Filed https://github.com/dart-lang/native/issues/1506